src/ophyd_async/core/signal_backend.py

```diff
@@ -45,3 +53,39 @@ async def get_setpoint(self) -> T:
     @abstractmethod
     def set_callback(self, callback: Optional[ReadingValueCallback[T]]) -> None:
         """Observe changes to the current value, timestamp and severity"""
+
+
+if TYPE_CHECKING:
+    SubsetEnum = Literal
+else:
+
```

Comment:
Actually, could we unindent the class definition and call it `class RuntimeSubsetEnum`, then do
```suggestion
if TYPE_CHECKING:
    SubsetEnum = Literal
else:
    SubsetEnum = RuntimeSubsetEnum
```
...

Reply:
Nice, stops the code being grayed out in the editor!